examples/VariantAndTelemetryDemo/Pages/Index.cshtml.cs

```diff
@@ -5,21 +5,15 @@
 
 namespace VariantAndTelemetryDemo.Pages
 {
-    public class IndexModel : PageModel
+    public class IndexModel(
+        IVariantFeatureManager featureManager,
+        TelemetryClient telemetry) : PageModel
     {
-        private readonly IVariantFeatureManager _featureManager;
-        private readonly TelemetryClient _telemetry;
+        private readonly IVariantFeatureManager _featureManager = featureManager ?? throw new ArgumentNullException(nameof(featureManager));
```

Comment:
Is this change triggered for any linting rule? I prefer the previous pattern.

Reply:
Yeah this was triggered by primary constructor: https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0290

Which I do think looks nicer- although it's a little less nice when we have ArgumentNullException checks

Reply:
> although it's a little less nice when we have ArgumentNullException checks

Yes. And I do think the previous one looks nicer to me. hahaha 

I don't remember our current linting rule set which only followed default settings of IDE0055 will enforce this. If not, I prefer to not this rule in the example projects.

Reply:
I don't feel strongly either way, so I'll go ahead and revert it.